c/tests/integration/driver_test.c

```diff
@@ -160,7 +160,7 @@ bool test_query_data() {
     bool success = false;
 
     // Set up connection & database
-    connection = connection_open_core(TYPEDB_CORE_ADDRESS);
+    connection = connection_open_core(TYPEDB_CORE_ADDRESS, DRIVER_LANG);
```

Comment:
Can we have two C entry points by any chance? Do we have a separate set of C functions that aren't just generated from Rust?

Eg. it would be nice to have `connection_open_core(address)` always include `"c"`, but then have another one `connection_open_core_with_description(address, lang)` which use from SWIG for java, python, etc.?

Then the users of the C driver don't have to say.... I'm using C! every time they connect :)

Also, these `connection` should  be `driver`, and the call should be `driver_open_core` or just `driver_core`

Reply:
I'll rename `connection`
I'll add a todo to change add a separate c entry point so we'll revisit it when we will be fixing the tests